packages/backend/migrations/20230202124709_create_ledger_transfers_table.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('ledgerTransfers', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.uuid('transferRef').notNullable().unique().index()
+
+    table.uuid('debitAccountId').notNullable()
+    table.foreign('debitAccountId').references('ledgerAccounts.id')
+    table.uuid('creditAccountId').notNullable()
+    table.foreign('creditAccountId').references('ledgerAccounts.id')
+
+    table.bigInteger('amount').notNullable().checkPositive()
+
+    table.uuid('assetId').notNullable()
+    table.foreign('assetId').references('assets.id')
+
+    table.enu('state', ['PENDING', 'POSTED', 'VOIDED', 'EXPIRED']).notNullable()
```

Comment:
I think we also need an optional `timeout`/expiry
https://github.com/tigerbeetledb/tigerbeetle/blob/main/docs/reference/transfers.md#timeout

That could maybe let us drop the `EXPIRED` state (a transfer would be expired is it's `PENDING` and the `createdAt`+`timeout` < now)